src/main/java/org/apache/maven/plugin/compiler/CompilerMojo.java

```diff
@@ -203,6 +203,18 @@ public void execute()
         }
     }
 
+    @Override
+    final Set<String> getIncludes()
```

Comment:
the usual pattern in the class is to make this open with `protected` and not `final`

Reply:
pattern followed